lib/encrypt.js

```diff
@@ -0,0 +1,128 @@
+/*!
+ * Copyright (c) 2021 Digital Bazaar, Inc. All rights reserved.
+ */
+import PouchDB from 'pouchdb';
+import pouchFind from 'pouchdb-find';
+
+// enable for debugging purposes only
+// import debugPouch from 'pouchdb-debug';
+// debugPouch(PouchDB);
+// PouchDB.debug.enable('pouchdb:find');
+
+PouchDB.plugin(pouchFind);
+
+export class EncryptedDb {
+  constructor({db, dbName, edv, invocationSigner}) {
+    this.edv = edv;
+    this.invocationSigner = invocationSigner;
+    this.db = db || new PouchDB(dbName);
+    // TODO: This needs to handle encrypted indexes.
```

Comment:
Issue has been created for this: https://github.com/digitalbazaar/bedrock-web-local-vc-store/issues/10